src/Modules/Installer/Console/InstallCommand.php

```diff
@@ -111,7 +112,8 @@ private function getInstallerConfiguration(): ?InstallerConfiguration
 
         if (!$this->configurationParser->configurationFileExists()) {
             $this->formatter->error(
-                'Please add the configuration file created by a previous install named fork-cms-installation-configuration.yaml before running the command in the root directory.'
+                'Please add the configuration file created by a previous install named
```

Comment:
Moet je hier niet met concat werken? Anders gaat die newline toch in de output komen?